salt/modules/kubernetes.py

```diff
@@ -2,7 +2,7 @@
 '''
 Module for handling kubernetes calls.
 
-:optdepends:    - kubernetes Python client
+:optdepends:    - kubernetes Python client up to version 2.0.0
```

Comment:
I would remove this `:optdepends:` change, since the current status of this PR also allows to use lower versions of kubernetes Python client.